.github/pull_request_template.md

```diff
@@ -9,12 +9,16 @@
 1. `x.java`
 2. `y.python`
 
+## 🚨 User Impact 🚨
+Are there any breaking changes? 
+What is the end result perceived by the user?
```

Comment:
I think it's probably more useful to put this in the connector README than in PR description? People usually won't check this.

Reply:
the goal of this is to get the developer to think about the end impact this particular PR they are raising will have on the user especially if it contains a breaking change. If there is a breaking change, then we want to make sure it is: 
1) very thoroughly considered
2) is communicated to users clearly perhaps via something like #8004 

this way, if there are any breaking issues or action required from the user, or just anything they really need to be aware of, this section will bring that to the forefront.

Reply:
Got it. Make sense.

I was actually thinking about the connector doc, but I wrote down "README" by mistake. It is probably also worth adding alert about breaking changes to the `CHANGELOG` section of the connector doc.